engine/benchmarks/src/main/scala/pl/touk/nussknacker/engine/benchmarks/avro/AvroSamples.scala

```diff
@@ -0,0 +1,35 @@
+package pl.touk.nussknacker.engine.benchmarks.avro
+
+import org.apache.avro.Schema
+import org.apache.avro.generic.GenericData
+import org.apache.flink.formats.avro.typeutils.LogicalTypesGenericRecordBuilder
+import pl.touk.nussknacker.engine.avro.AvroUtils
+
+object AvroSamples {
+
+  val sampleSchemaId = 123
+
+  private val fieldsCount = 10
+
+  val sampleSchema: Schema = AvroUtils.parseSchema(s"""{
```

Comment:
Can't we use Schema.Builder instead of raw strings? :)

Reply:
raw string is easier to copy-paste from somewhere :)